code/game/gamemodes/miniantags/bot_swarm/swarmer.dm

```diff
@@ -260,14 +260,19 @@
 	S.DisIntegrate(src)
 	return TRUE
 
-/obj/machinery/door/swarmer_act(mob/living/simple_animal/hostile/swarmer/S)
+/obj/machinery/door/swarmer_act(mob/living/simple_animal/hostile/swarmer/S, var/datum/gas_mixture/air_contents)
 	var/isonshuttle = istype(get_area(src), /area/shuttle)
 	for(var/turf/T in range(1, src))
+		var/datum/gas_mixture/pressure_check = T.return_air()
 		var/area/A = get_area(T)
 		if(isspaceturf(T) || (!isonshuttle && (istype(A, /area/shuttle) || istype(A, /area/space))) || (isonshuttle && !istype(A, /area/shuttle)))
 			to_chat(S, "<span class='warning'>Destroying this object has the potential to cause a hull breach. Aborting.</span>")
 			S.target = null
-			return FALSE
+			return TRUE
+		else if(pressure_check.return_pressure() > 5000)  //If pressure > 5000kPa
```

Comment:
Magic number here. Best to make use of a define or make one yourself. `ONE_ATMOSPHERE` for instance is the define for the value of one atmosphere. Found in __DEFINES/atmospherics.dm